src/biogeochem/DUSTMod.F90

```diff
@@ -356,86 +565,222 @@ subroutine DustEmission (bounds, &
+         if (lnd_frc_mbl(p) > 0.0_r8) then  ! if bare land fraction is larger than 0 then calculate the dust emission equation
 
             ! reset these variables which will be updated in the following if-block
 
-            wnd_frc_slt = fv(p)
+            !wnd_frc_slt = fv(p)  ! we don't need this line because its calculation is moved upward
             flx_mss_hrz_slt_ttl = 0.0_r8
             flx_mss_vrt_dst_ttl(p) = 0.0_r8
 
             ! the following line comes from subr. dst_mbl
             ! purpose: threshold saltation wind speed
 
-            wnd_rfr_thr_slt = u10(p) * wnd_frc_thr_slt / fv(p)
-
-            ! the following if-block comes from subr. wnd_frc_slt_get 
-            ! purpose: compute the saltating friction velocity
-            ! theory: saltation roughens the boundary layer, AKA "Owen's effect"
-
-            if (u10(p) >= wnd_rfr_thr_slt) then
-               wnd_rfr_dlt = u10(p) - wnd_rfr_thr_slt
-               wnd_frc_slt_dlt = 0.003_r8 * wnd_rfr_dlt * wnd_rfr_dlt
-               wnd_frc_slt = fv(p) + wnd_frc_slt_dlt
-            end if
+            wnd_rfr_thr_slt = u10(p) * wnd_frc_thr_slt / fv(p)     ! keep and use if we want the default Z03 scheme
 
+            
             ! the following comes from subr. flx_mss_hrz_slt_ttl_Whi79_get
             ! purpose: compute vertically integrated streamwise mass flux of particles
 
-            if (wnd_frc_slt > wnd_frc_thr_slt) then
-               wnd_frc_rat = wnd_frc_thr_slt / wnd_frc_slt
-               flx_mss_hrz_slt_ttl = cst_slt * forc_rho(c) * (wnd_frc_slt**3.0_r8) * &
-                    (1.0_r8 - wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) / grav
+            !if (wnd_frc_slt > wnd_frc_thr_slt) then! if using Zender's scheme, use fluid threshold for dust emission equation below
+            if (wnd_frc_slt > wnd_frc_thr_slt_it) then! if using Leung's scheme, use impact threshold for dust emission equation
+
+               !################### for Zender et al. (2003) scheme -dmleung ###########################
+               !################ uncomment the below block if want to use Z03 scheme ###################
+               !wnd_frc_rat = wnd_frc_thr_slt / wnd_frc_slt
+               !flx_mss_hrz_slt_ttl = cst_slt * forc_rho(c) * (wnd_frc_slt**3.0_r8) * &
+               !     (1.0_r8 - wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) / grav  ! Zender dust emission equation for emission flux
 
                ! the following loop originates from subr. dst_mbl
                ! purpose: apply land sfc and veg limitations and global tuning factor
                ! slevis: multiply flx_mss_hrz_slt_ttl by liqfrac to incude the effect 
                ! of frozen soil
 
-               flx_mss_hrz_slt_ttl = flx_mss_hrz_slt_ttl * lnd_frc_mbl(p) * mbl_bsn_fct(c) * &
-                    flx_mss_fdg_fct * liqfrac
+               !flx_mss_hrz_slt_ttl = flx_mss_hrz_slt_ttl * lnd_frc_mbl(p) * mbl_bsn_fct(c) * &
```

Comment:
Remove commented out code.

Reply:
This part is Zender's dust emission scheme, so we can probably keep and add a switch to switch between Leung and Zender...

Reply:
What I'm planning on doing is to remove it here and have a separate module for the Zender scheme and the Leung scheme. This will be built with OOP having a base class that extends into either the Zender version or the Leung version. In the future other methods could be added, without having to have explicit if statements in the code.

Reply:
And I did remove it here.